matcher/service/routes.py

```diff
@@ -18,6 +18,13 @@ class MatcherStatusException(Exception):
     '''Exception wrapper class for errors related to the status of the Matcher'''
     pass
 
+def set_config_status(openreview_client, config_note_id, status, message=''):
```

Comment:
isn't `ConfigNoteInterface` in charge of this in `set_status` function? why do you create a new implementation?